src/adr/0030-separate-attachment-content-and-metadata.md

```diff
@@ -60,7 +63,10 @@ Some adjustments are needed on class working with attachment:
  attachment should be parsed or not on an implementation aware fashion, saving attachment parsing upon writes for JPA 
  and Maildir.
  
-Maildir and JPA no longer support attachment content loading.
+Maildir and JPA no longer support attachment content loading. Only the JMAP protocol requires attachment content loading,
+which is not supported on top of these technologies. Attachment loading could be supported in the future via coupling on 
```

Comment:
I don't get why we would describe a solution here (that I don't understand as-is anyway)

Reply:
Then I will remove the explanation